src/reefState/providerState.ts

```diff
@@ -22,6 +23,11 @@ import {
 import { Network } from "../network/network";
 import { forceReload$ } from "./token/force-reload-tokens";
 
+export const providerSubj: ReplaySubject<Provider> =
```

Comment:
this is not needed provider is instantiated from initReefState and available as selectedProvider$ https://github.com/reef-chain/util-lib/blob/5226cd9dac56118d9a09fc25531501b2c7383adb/src/reefState/providerState.ts#L102